scripts/decredStakeVoting.md

```diff
@@ -1,46 +1,47 @@
 
-First, each ticket votes on the validity of the last seen block. If a block is deemed invalid by at least a 3 to 5 majority, then the entire block is invalidated except for the proof-of-stake portion of the block reward. This means that the proof-of-work reward and the transactions are stripped from the network. This process acts as a check on miner power, giving stakeholders a way to crack down on miners who are not following the rules and prevent 51% attacks.
+When a ticket is drawn, it's owner's vote will be cast by their configured stake server.
 
-Secondly, a ticket may cast `yes`, `no`, or `abstain` votes concerning upgrade proposals submitted by stakeholders through Politeia; Decred’s proposal system.
-If a specific proposed update receives the required number of `yes` votes, it is collectively implemented by the network and upgrades the consensus rules of the chain. This process puts the power to decide the future of Decred in the hands of stakeholders and prevents potentially troublesome chain splits.
-Once a ticket has voted, the Decred spent to purchase the ticket is unlocked and a reward for voting, called a ticket reward, is deposited in the ticket owners account. This ticket reward amounts to 6% of the total decred block reward. After a waiting period of 20 hours, these funds are available to be spent on additional tickets.
+Stake servers allow the casting of votes without worry of losses due to computer downtime or bad Internet connections. In return, stake servers receive a small fee taken from vote proceeds. This server fee helps pay for the cost of server infrastructure needed to ensure stake votes are cast quickly and reliably.
 
-Stake voting requires no action by the stakeholder other than purchasing tickets. By default, tickets are configured to abstain from voting on upgrade proposals, however `yes` or `no` votes can be configured easily through the Decrediton staking wallet.
+The 5 stake votes contained in each new block vote on the validity of the previous one. If this block is deemed invalid by at least a 3 to 5 majority, then its associated transactions and proof-of-work reward are stripped from the network while the proof-of-stake reward is retained. This process acts as a check on miner power, giving stakeholders a way to crack down on miners who are not following the rules and prevent miner aggressions such as block withholding or a 51% attack.
 
-In order to ensure that a vote is not missed, a stake server can be used.
-Stake servers allow stake voters to cast votes without worrying about missed votes due to computer downtime, bad internet connections, or inconsistent electricity grids.
+A ticket may also cast votes concerning other agenda items put up for a community vote. One example of an item that was approved in this manner was a consensus rule upgrade improving the algorithm which sets ticket price.
```

Comment:
Tickets also vote concerning any other agenda items that are up for community vote, such as any concensus rule upgrades being proposed. All of the code needed for the upgrade will already be built into the latest version of the software, and the Decred stake holders approve the upgrade. This was how the rules were upgraded to improve the efficiency of the ticket pricing algorithm, its how decred will get upgraded to be compatible with lightning network, and it can also be used for if the community wants to change the size of the blocks.